tests/Action/SetObjectFieldValueActionTest.php

```diff
@@ -126,7 +126,25 @@ public function testSetObjectFieldValueAction(): void
         $this->assertSame(Response::HTTP_OK, $response->getStatusCode());
     }
 
-    public function testSetObjectFieldValueActionWithDate(): void
+    public function getTimeZones(): iterable
+    {
+        $default = new \DateTimeZone(date_default_timezone_get());
+        $custom = new \DateTimeZone('Europe/Rome');
+
+        return [
+            [null, $default],
```

Comment:
Bonus points if you create named datasets:
```suggestion
            'empty timezone' => [null, $default],
```

Reply:
Thanks. I added this.